docker/Dockerfile

```diff
@@ -8,16 +8,28 @@ RUN mkdir -p /opt/zoraxy/source/ &&\
 
 RUN chmod -R 770 /opt/zoraxy/
 
-VOLUME [ "/opt/zoraxy/config/" ]
-
 # If you build it yourself, you will need to add the src directory into the docker directory.
 COPY ./src/ /opt/zoraxy/source/
 
 WORKDIR /opt/zoraxy/source/
 
 RUN go mod tidy &&\
-    go build -o /usr/local/bin/zoraxy &&\
-    rm -r /opt/zoraxy/source/
+go build -o /usr/local/bin/zoraxy &&\
```

Comment:
Slight nit:
Would you mind tabbing these? Just for continuity. Same further down.

Reply:
Sure, i'll do that 👍

Reply:
Sorry for the long wait.
I now formated the Dockerfile @PassiveLemon